plugins/base/src/main/resources/dokka/scripts/symbol-parameters-wrapper_deferred.js

```diff
@@ -1,83 +1,57 @@
-        const symbolElement = entry.target
-        symbolResizeObserver.unobserve(symbolElement) // only need it once, otherwise will be executed multiple times
-        wrapSymbolParameters(symbolElement);
-    })
-});
-
-const wrapAllSymbolParameters = () => {
-    document.querySelectorAll("div.symbol").forEach(symbol => wrapSymbolParameters(symbol))
-}
-
-const wrapSymbolParameters = (symbol) => {
-    let parametersBlock = symbol.querySelector("span.parameters")
-    if (parametersBlock == null) {
-        return // nothing to wrap
-    }
-
-    let symbolBlockWidth = symbol.clientWidth
-
-    // Even though the script is marked as `defer` and we wait for `DOMContentLoaded` event,
-    // it can happen that `symbolBlockWidth` is 0, indicating that something hasn't been loaded.
-    // In this case, just retry once all styles have been applied and it has been resized correctly.
-    if (symbolBlockWidth === 0) {
-        symbolResizeObserver.observe(symbol)
-        return
+(function() {
+    const leftPaddingPx = 60;
+
+    function createNbspIndent () {
+        let indent = document.createElement("span")
+        indent.append(document.createTextNode("\u00A0\u00A0\u00A0\u00A0"))
+        indent.classList.add("nbsp-indent")
+        return indent
     }
 
-    let innerTextWidth = Array.from(symbol.children)
-        .filter(it => !it.classList.contains("block")) // blocks are usually on their own (like annotations), so ignore it
-        .map(it => it.getBoundingClientRect().width).reduce((a, b) => a + b, 0)
-
-    // if signature text takes up more than a single line, wrap params for readability
-    let shouldWrapParams = innerTextWidth > (symbolBlockWidth - leftPaddingPx)
-    if (shouldWrapParams) {
-        parametersBlock.classList.add("wrapped")
-        parametersBlock.querySelectorAll("span.parameter").forEach(param => {
-            // has to be a physical indent so that it can be copied. styles like
-            // paddings and `::before { content: "    " }` do not work for that
-            param.prepend(createNbspIndent())
-        })
+    function wrapSymbolParameters(entry) {
+        const symbol = entry.target;
+        const futureWidth = entry.borderBoxSize && entry.borderBoxSize[0] && entry.borderBoxSize[0].inlineSize;
+
+        // Even though the script is marked as `defer` and we wait for `DOMContentLoaded` event,
+        // or if this block is a part of hidden tab, it can happen that `futureWidth` is 0,
+        // indicating that something hasn't been loaded.
+        // In this case, observer will be triggered one again when it will be ready.
+        if (futureWidth > 0) {
+            const node = symbol.querySelector(".parameters")
+
+            if (node) { // need to re-calculate if params need to be wrapped after resize
+                node.classList.remove("wrapped")
+                node.querySelectorAll(".parameter .nbsp-indent")
+                    .forEach(indent => indent.remove());
```

Comment:
The comment migrated from the old code, and it seems to be misleading now - it doesn't actually check if the params need to be re-wrapped after resize (like it did before with `window.onresize`), the reset will happen anyway, right?

Is the comment from the suggestion correct?

```suggestion
            if (node) {
            
                // if window resize happened and observer was triggered, reset previously wrapped
                // parameters as they might not need wrapping anymore, and check again
                node.classList.remove("wrapped")
                node.querySelectorAll(".parameter .nbsp-indent")
                    .forEach(indent => indent.remove());
```

Reply:
✅